fedimint-testing/src/btc/mock.rs

```diff
@@ -243,11 +244,11 @@ impl IBitcoindRpc for FakeBitcoinTest {
     }
 
     async fn get_block_height(&self) -> BitcoinRpcResult<u64> {
-        Ok(self.blocks.lock().unwrap().len() as u64)
+        Ok((self.blocks.lock().unwrap().len() as u64) - 1)
```

Comment:
I'm confused. If we have 0 blocks, the "block height" is `0`, no? Not `-1`.

Reply:
Ohhhhhhhhh...


We should really name this method `get_block_count` like in Bitcoin Core.